yt_dlp/YoutubeDL.py

```diff
@@ -318,7 +318,7 @@ class YoutubeDL(object):
                                            about it, warn otherwise (default)
     source_address:    Client-side IP address to bind to.
     call_home:         Boolean, true iff we are allowed to contact the
-                       youtube-dlc servers for debugging.
+                       yt-dlp servers for debugging.
```

Comment:
```suggestion
                       yt-dlp servers for debugging. (BROKEN)
```

Since we are editing this anyway, why not add this :D